spec/features/orders/list_orders_spec.rb

```diff
@@ -292,4 +319,56 @@
       expect(order_list_table_cols).to include('Code')
     end
   end
+
+  feature 'with ordering' do
+    let!(:support_request_order_for_ordering)  { FactoryGirl.create(:order,
+                                                                    :order_support_request_for_ordering,
+                                                                    order_type: support_request_type) }
+    let(:order_2) { ['ORD-2', 'Support request', 'In progress',
+                     in_current_locale(support_request_order.created_at), '', 'support_ext_code', '',
+                     in_current_locale(support_request_order.estimated_exec_date)] }
+
+    let(:order_3) { ['ORD-3', 'Support request', 'In progress',
+                     in_current_locale(support_request_order_for_ordering.created_at), '', 'support_ext_code', '',
+                     in_current_locale(support_request_order_for_ordering.estimated_exec_date)] }
+
+    scenario 'by default fields' do
+      # "Order type"
+      change_select2_value('order_type_id', 'Support request')
+      search_button.click
+      wait_for_ajax
+      expect(orders_list).to eq([[], order_3, order_2])
+
+      order_list_table_header('Code').click
+      expect(orders_list).to eq([[], order_2, order_3])
+
+      order_list_table_header('Code').click
+      expect(orders_list).to eq([[], order_3, order_2])
+    end
+
+    scenario 'by custom fields' do
+      order_2_with_contract = order_2 + [support_request_order.data['contractNumber'].to_s]
+      order_3_with_contract = order_3 + [support_request_order_for_ordering.data['contractNumber'].to_s]
+
+      # "Order type"
+      change_select2_value('order_type_id', 'Support request')
+      search_button.click
+      wait_for_ajax
+      expect(orders_list).to eq([[], order_3, order_2])
+
+      # Add Contract number to orders table
+      custom_fields_button.click
+      custom_fields_list('Contract number').click
+      custom_fields_button.trigger('click')
```

Comment:
But we already have `click_on_multiselect_options(id, options)`, why didn't you use it?